operator/helper/time.go

```diff
@@ -129,6 +139,28 @@ func (t *TimeParser) Parse(ctx context.Context, entry *entry.Entry) error {
 	return nil
 }
 
+func (t *TimeParser) parseUnixHexTime(value interface{}) (time.Time, error) {
+	var ti string
+
+	switch v := value.(type) {
+	case []byte:
+		ti = string(v)
+	case string:
+		ti = fmt.Sprintf("%v", v)
+	case int, int32, int64, uint32, uint64:
+		ti = fmt.Sprintf("%d", v)
```

Comment:
I don't think it makes sense to convert integer values to their decimal representation before parsing them as hex. It's probably safer to just not handle these values and return an error in the default branch since they're not actually a unixhex time

Reply:
You're right I mistakenly wrote this to pass the test of `time.Now().Unix()` but I change the rule.